src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IObjectCreationExpression.vb

```diff
@@ -60,15 +60,15 @@ IBlockStatement (9 statements, 7 locals) (OperationKind.BlockStatement, IsInvali
       Variables: Local_1: x3 As F
       Initializer: IObjectCreationExpression (Constructor: Sub F..ctor()) (OperationKind.ObjectCreationExpression, Type: F) (Syntax: 'New F() Wit ... erty1 = ""}')
           Initializers(1): IAssignmentExpression (OperationKind.AssignmentExpression, Type: System.Void) (Syntax: '.Property1 = ""')
-              Left: IIndexedPropertyReferenceExpression: Property F.Property1 As System.String (OperationKind.PropertyReferenceExpression, Type: System.String) (Syntax: 'Property1')
+              Left: IPropertyReferenceExpression: Property F.Property1 As System.String (OperationKind.PropertyReferenceExpression, Type: System.String) (Syntax: 'Property1')
```

Comment:
@heejaechang Yes, I thought I had fixed these up in features/IOperation, but master still has the wrong expected dump for these tests.